test/integration/targets/setup_paramiko/install-FreeBSD-11.4-python-3.yml

```diff
@@ -0,0 +1,3 @@
+- name: Install Paramiko for Python 3 on FreeBSD 11.4
+  pkgng:
+    name: py37-paramiko
```

Comment:
Shouldn't this use Python 3.6 instead, since that's the version we list in the completion file?

The same would apply to the other file below.